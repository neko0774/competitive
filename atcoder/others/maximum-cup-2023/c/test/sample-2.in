2000000000 3
20 23 7
