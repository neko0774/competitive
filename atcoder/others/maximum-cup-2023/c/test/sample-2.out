1597222223653885214
